pkgs/os-specific/darwin/macmon/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "macmon";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "vladkens";
+    repo = "macmon";
+    rev = "v${version}";
+    hash = "sha256-COpEjK1LUwGzhSgD09D4gx+MtS2hT0qt06rTPT8JQiQ=";
+  };
+
+  cargoHash = "sha256-DTkpFGl8kTWttFGKTCpny2L0IRrCgpnnXaKIFoxWrW4=";
+
+  buildInputs = [ darwin.apple_sdk.frameworks.IOKit ];
```

Comment:
```suggestion
```

Not needed anymore, as `apple-sdk` is implicitly added to the `stdenv` when on Darwin (all frameworks are directly available).